src/pages/dashboard/Dashboard.js

```diff
@@ -0,0 +1,19 @@
+import {useAuth0} from "@auth0/auth0-react";
+import {Button} from "@mui/material";
+import "./Dasboard.css";
+
+function Dashboard() {
+  const {logout, user, isAuthenticated} = useAuth0();
+  return (
+    isAuthenticated &&
+      <div className="dashboard">
+    <Button variant="contained" size="large" onClick={() => logout()}>
```

Comment:
No need to create anonymous arrow function if not passing any params
```suggestion
    <Button variant="contained" size="large" onClick={logout}>
```